README.md

```diff
@@ -1,2 +1,49 @@
-# cmr_geometry_utils
-CMR geometry utils library
+<div align="center">
+
+# CMR Tests Utils
```

Comment:
but the repo is called `cmr_geometry_utils`?  confused

Reply:
oops readme isn't updated